TrafficSimulator/Assets/Emmision.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections;
```

Comment:
Rename file to `Emission.cs` or remove since it is empty and add it later when it is used